src/engine/channels/enginedeck.h

```diff
@@ -25,7 +25,16 @@ class EngineDeck : public EngineChannel, public AudioDestination {
 
     void process(CSAMPLE* pOutput, const int iBufferSize) override;
     void collectFeatures(GroupFeatureState* pGroupFeatures) const override;
+
+    // postProcessLocalBpm() is called on all decks to update the localBpm after
+    // process() is done. Updated localBpm for all decks are required for the
+    // postPrecess() step, to not deal with the order they are processed.
+    // It cannot be done during process() because it relies that the localBpm
+    // of all decks are on their old values.
```

Comment:
slight language tweak

```suggestion
    // postProcessLocalBpm() is called on all decks to update the localBpm after
    // process() is done. Updated localBpms for all decks are required for the
    // postProcess() step, to avoid issues with the order they are processed.
    // It cannot be done during process() because it relies that the localBpm
    // of all decks are on their old values.
```